src/Installers/Windows/WindowsHostingBundle/thm.xml

```diff
@@ -1,13 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Theme xmlns="http://wixtoolset.org/schemas/thmutil/2010">
+﻿<Theme xmlns="http://wixtoolset.org/schemas/v4/thmutil">
     <Window Width="485" Height="347" HexStyle="100a0000" FontId="0">#(loc.Caption)</Window>
     <Font Id="0" Height="-12" Weight="500" Foreground="000000" Background="FFFFFF">Segoe UI</Font>
     <Font Id="1" Height="-24" Weight="500" Foreground="000000">Segoe UI</Font>
     <Font Id="2" Height="-22" Weight="500" Foreground="666666">Segoe UI</Font>
     <Font Id="3" Height="-12" Weight="500" Foreground="000000" Background="FFFFFF">Segoe UI</Font>
     <Font Id="4" Height="-12" Weight="500" Foreground="ff0000" Background="FFFFFF" Underline="yes">Segoe UI</Font>
```

Comment:
Take a look at the SDK PR - you should consider using the predefined system colors. Those now work with high contrast themes, making the installer pass a number of accessibility tests that failed in v3